examples/stable-diffusion/README.md

```diff
@@ -294,3 +294,31 @@ image = pipe(prompt, num_inference_steps=50, guidance_scale=7.5).images[0]
 
 image.save("cat-backpack.png")
 ```
+
+
+## Fine-Tuning
+
+The `train_text_to_image.py` script shows how to implement the fine-tuning of Stable Diffusion models on Habana Gaudi.
+
+### Example for SDXL
+We can launch the fine-tuning of SDXL model using:
+
+python training/train_text_to_image.py \
+  --pretrained_model_name_or_path stabilityai/stable-diffusion-xl-base-1.0 \
+  --pretrained_vae_model_name_or_path stabilityai/sdxl-vae \
+  --dataset_name lambdalabs/pokemon-blip-captions \
+  --resolution 1024 \
```

Comment:
I think this was typo from original SD.  Stable Diffusion XL was trained at a base resolution of 1024x1024 and SDXL is operating on 1024 as default resolution.  If you look at added LoRA command for training it was fixed to 1024:  https://github.com/huggingface/diffusers/blob/main/examples/text_to_image/README_sdxl.md#training-1

Therefore, I would leave 1024 for fine-tuning SDXL.   If we want to copy the diffusers README command then I can update, but IMO it would not be correct.